### PR TITLE
#7045 | Adding Acquia as vendor

### LINKED ISSUE #7045: Intent to Implement: amp-analytics support for Acquia
Acquia intends to implement analytics support with AMP. I am not sure if we need to add more details. We are currently discussing internally what to do with https://github.com/ampproject/amphtml/issues/6031 so this is somewhat an impediment for us to support AMP quickly.

### PR DESCRIPTION
**Pull request by nickveenhof**
This is the PR following the intent to implement for the Acquia Lift analytics service. See issue #7045 

This PR is not complete yet and still has to be tested on our end. I do welcome reviews already so that we can start putting every detail in the right place. I added our vendor to the top of the list as it seemed to be alphabetically ordered, happy to change that if this was not intended.

Closes #7045.

### CODE REVIEW

**Comment by nickveenhof** [comment]
Reviews welcome. Tests passing. Please hold with merging it till we confirm everything on our end

**Comment by nickveenhof** [comment]
Pinging @avimehta for initial review

File: examples/analytics-vendors.amp.html [hunk 1]
```diff
@@ -27,6 +27,44 @@
 <div id="container">
 Container for analytics tags. Positioned far away from top to make sure that doesn't matter.
 
+<!-- Acquia Lift example -->
+<amp-analytics type="acquialift" id="acquialift">
++<!-- acquialift type -->
```
Comment (avimehta) [inline examples/analytics-vendors.amp.html]: There see to be two + here. Is that correct? Also, the comment seems redundant (or I do the know what it is for). Maybe remove it?

Comment (nickveenhof) [inline examples/analytics-vendors.amp.html]: Resolved

**Comment by nickveenhof** [comment]
Still testing internally. Please hold for a bit longer before merging in. Thanks

**Comment by nickveenhof** [comment]
@avimehta, we're good to go with this PR. Please merge whenever you can/want.

File: examples/analytics-vendors.amp.html [hunk 2]
```diff
@@ -27,6 +27,43 @@
...
+    "siteId":"my-acquia-lift-site-id",
+    "accountId": "MYACCOUNTID",
+    "decisionApiUrl": "us-east-1-decisionapi.lift.acquia.com"
+  },
+  "triggers": {
+    "pageLoad": {
+      "on": "visible",
+      "request": "pageview"
+    },
+    // Sends a Click-Through event.
```
Comment (avimehta) [inline examples/analytics-vendors.amp.html]: Sorry for nitpick, These comments will make the json parsing to fail. To make sure everythign is working, have you also done 
`gulp` and then gone to /examples/analytics-vendors.amp.html?

Comment (nickveenhof) [inline examples/analytics-vendors.amp.html]: I did. did gulp watch and gulp serve to test the actual requests. Removed the comments. Thanks for the nitpick. Always appreciated :)

**Comment by nickveenhof** [comment]
@avimehta, can you please verify again? thanks!

**Pull request merged**